ios/ci_scripts/ci_post_clone.sh

```
@@ -1,15 +1,93 @@
+PROJECT_DIR="$(pwd)/.."  # Goes up one level from ci_scripts to ios directory
+
+echo "PROJECT_DIR: $PROJECT_DIR"
+echo "Info.plist path: $PROJECT_DIR/Cypherd/Info.plist"
+
+# Get current version from Info.plist
+CURRENT_VERSION=$(/usr/libexec/PlistBuddy -c "Print :CFBundleShortVersionString" "$PROJECT_DIR/Cypherd/Info.plist")
+CURRENT_BUILD=$(/usr/libexec/PlistBuddy -c "Print :CFBundleVersion" "$PROJECT_DIR/Cypherd/Info.plist")
+
+# Increment version if target branch is main
+if [ "$CI_TARGET_BRANCH" = "main" ]; then
+    IFS='.' read -ra VERSION_PARTS <<< "$CURRENT_VERSION"
```

Shellcheck (0.9.0-1)

[warning] In POSIX sh, read -a is undefined.

(SC3045)